tests/k6/tests/serviceowner/authentication.js

```diff
@@ -66,16 +66,6 @@ export default function () {
             expectation(r);
         });
 
-        describe(`Attempt dialog element get ${message} token`, () => {
-            let r = getSO("dialogs/" + uuidv4() + "/elements/" + uuidv4(), params);
-            expectation(r);
-        });
-
-        describe(`Attempt dialog element post ${message} token`, () => {
-            let r = postSO("dialogs/" + uuidv4() + "/elements/", {}, params);
-            expectation(r);
-        });
-
```

Comment:
@elsand 
Will transmissions have their own GET/SEARCH endpoints as Elements had? 
(So we could comment these out so we remember to re-enable when doing the transmissions implementation)

Reply:
Yes.